### PR TITLE
Temporarily fetch refdata from GDrive

### PR DESCRIPTION
**Pull request by epassaro**
## Description
Retrieve TARDIS reference data from a backup stored in Google Drive.

## Motivation and Context

We exceeded our LFS quota by mistake :(

## How Has This Been Tested?

Testing pipeline is passing.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have assigned/requested two reviewers for this pull request.

### CODE REVIEW

File: azure-pipelines/simple_test_framework.yml [hunk 1]
```diff
@@ -44,30 +44,37 @@ jobs:
...
+          cd $(ref.data.home)
+          git fetch origin
+          git checkout origin/master
+          git lfs pull --include="atom_data/kurucz_cd23_chianti_H_He.h5" origin
+          git lfs pull --include="atom_data/chianti_He.h5" origin
+          git lfs pull --include="unit_test_data.h5" origin
+          git lfs pull --include="packet_unittest.h5" origin
+          echo MD5: `md5sum unit_test_data.h5`
         displayName: "Fetch reference data"
+        condition: false
```
Comment (epassaro) [inline azure-pipelines/simple_test_framework.yml:63]: I just removed the `plasma_reference` folder (does not exist in tardis-sn/tardis-refdata repo) and disabled the step with the `false` condition. The rest of the lines were moved by the YAML autoformatter.

File: azure-pipelines/simple_test_framework.yml [hunk 2]
```diff
@@ -44,30 +44,37 @@ jobs:
...
+        displayName: "Fetch reference data (GDrive backup)"
 
       - bash: |
-              source activate tardis
-              pip install pytest-azurepipelines
-              pytest tardis --tardis-refdata=$(ref.data.home) --cov=tardis --cov-report=xml --cov-report=html
+          source activate tardis
+          pip install pytest-azurepipelines
+          pytest tardis --tardis-refdata=$(ref.data.home) --cov=tardis --cov-report=xml --cov-report=html
         displayName: "TARDIS test"
```
Comment (epassaro) [inline azure-pipelines/simple_test_framework.yml:76]: No changes here, just the YAML autoformatter.

File: azure-pipelines/simple_test_framework.yml [hunk 3]
```diff
@@ -44,30 +44,37 @@ jobs:
...
 
       - bash: |
-              bash <(curl -s https://codecov.io/bash)
+          bash <(curl -s https://codecov.io/bash)
```
Comment (epassaro) [inline azure-pipelines/simple_test_framework.yml:79]: YAML autoformatter.

File: azure-pipelines/simple_test_framework.yml [hunk 4]
```diff
@@ -44,30 +44,37 @@ jobs:
...
-              source activate tardis
-              pip install pytest-azurepipelines
-              pytest tardis --tardis-refdata=$(ref.data.home) --cov=tardis --cov-report=xml --cov-report=html
+          source activate tardis
+          pip install pytest-azurepipelines
+          pytest tardis --tardis-refdata=$(ref.data.home) --cov=tardis --cov-report=xml --cov-report=html
         displayName: "TARDIS test"
 
       - bash: |
-              bash <(curl -s https://codecov.io/bash)
```
Comment (epassaro) [inline azure-pipelines/simple_test_framework.yml:72]: YAML autoformatter.

File: azure-pipelines/simple_test_framework.yml [hunk 5]
```diff
@@ -44,29 +44,37 @@ jobs:
...
       - bash: |
-              source activate tardis
-              python setup.py build_ext --inplace
+          source activate tardis
+          pip install gdown
+          gdown https://drive.google.com/uc\?id\=1LwfZ6MXuV2j9a23uu9HTcoDDFI9gIuf1 -O tardis-refdata.tar.gz
+          tar zxf tardis-refdata.tar.gz
+          mv tardis-refdata $(ref.data.home)
+        displayName: "Fetch reference data (GDrive backup)"
+
```
Comment (wkerzendorf) [inline azure-pipelines/simple_test_framework.yml:66]:
```suggestion
        condition: true
```

**Pull request merged**